namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_route53_zone" "example_team_route53_zone" {
+  name = var.zone-name
+
+  tags = {
+    application            = var.application
+    is-production          = var.is-production
+    environment-name       = var.environment-name
+    owner                  = var.team_name
+    infrastructure-support = var.infrastructure-support
+  }
+}
+
+resource "kubernetes_secret" "example_route53_zone_sec" {
+  metadata {
+    name      = "example-route53-zone-output"
+    namespace = var.namespace
+  }
+
+  data = {
+    zone_id = aws_route53_zone.example_team_route53_zone.zone_id
+  }
+}
+
+resource "aws_route53_record" "add_cname_email" {
+  name    = var.zone-name
+  zone_id = aws_route53_zone.example_team_route53_zone.zone_id
+  type    = "CNAME"
+  records = ["test.org"]
+  ttl     = "300"
+}
```

Comment:
i believe you don't need this- not a blocker though

Reply:
And it won't hurt ?

Reply:
It'd be better to delete it

Reply:
Done